Cargo.toml

```diff
@@ -47,7 +47,7 @@ flate2 = { version = "1.0.30", default-features = false, features = ["zlib"] }
 git2 = "0.18.3"
 git2-curl = "0.19.0"
 gix = { version = "0.62.0", default-features = false, features = ["blocking-http-transport-curl", "progress-tree", "revision", "parallel", "dirwalk"] }
-glob = "0.3.1"
+glob = { git = "https://github.com/rust-lang/glob.git", rev = "47c0cf3d03c56608f4371a3a832ec752c1af328f" }
```

Comment:
Since we publish the `cargo` package, we can't rely on git submodulese

Reply:
Okay~ I got it~